### DISCUSSION TITLE
Add RustDesk to Flutter showcase?

### DISCUSSION THREAD

**Discussion by fzyzcjy** [rustdesk/rustdesk#1849] [General]
Hi, do you have interest in being in the "showcase" of flutter.dev? This one:

![image](https://user-images.githubusercontent.com/5236035/198186270-b902c9eb-3c37-4a9d-a756-b5ca59ee5c42.png)

I have sent message to Flutter discord as well (see below). I am not sure whether this is possible, but I guess maybe Flutter showcase did not know RustDesk and RustDesk did not know Flutter showcase previously. Thus I post to both sides :)

(P.S. that channel is publicly available, but you may need a few configuration steps before entering that channel)

https://discord.com/channels/608014603317936148/608018585025118217/1035035504061722694

![image](https://user-images.githubusercontent.com/5236035/198186120-f81f6eb7-a668-438f-8aeb-fc2e5bb8925b.png)

**Comment by rustdesk** [comment]
We have interest. Thanks a lot.

**Comment by fzyzcjy** [comment]
You are welcome!

**Comment by fzyzcjy** [comment]
@rustdesk Not sure whether you have seen this, anyway here is a copy of the public discussion as a synchronization

![image](https://user-images.githubusercontent.com/5236035/198411513-240dd223-5574-4eb7-85fc-e05ce265c1a1.png)
![image](https://user-images.githubusercontent.com/5236035/198412041-6edcfc27-dbfa-445b-a2c0-ee975b77b766.png)

**Comment by fzyzcjy** [comment]
@rustdesk So maybe you have to find out how to contact them... I have no idea about that since I major in programming not in marketing

**Reply by rustdesk** [reply]
forget it, i hate to contact google, it is only wasting time. 

Thanks a lot, you have done a lot.

**Reply by fzyzcjy** [reply]
@rustdesk You are welcome.

> i hate to contact google, it is only wasting time.

I feel flutter tech team quite good for tech problems (e.g. reviewing PRs) - never contacted marketing teams though
